src/cache_refresh/cache_refresh.cc

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <asm-generic/errno-base.h>
```

Comment:
why was this needed?

Reply:
I'm not sure how that include got in there, removed it.